ivy-scaling-nginx/nginx.conf

```diff
@@ -0,0 +1,46 @@
+events {
+  worker_connections  1024;
+}
+
+http {
+  upstream ivy-cluster {
+    ip_hash;
+    server ivy:8080;
```

Comment:
if you scale up. you don't have to modify here something?

Reply:
Interesting enough, no, it works perfectly this way.

Reply:
Seems this works only if you start more than one instance at startup. If you start only one instance at startup. Then scale up and then kill first instance I still get 502 errors. Seems in this case NGNIX does not understand that there are mutliple IPs behind the DNS entry. We should therefore scale to 2 by default.